### PR TITLE
fix: governance widgets CSS

### PR DESCRIPTION
**Pull request by DiogoSoaress**
## What it solves
Fixes some styles errors. List:
- [x] collapsible section area only clickable in the icon
- [ ] ~reload the Safe app on error~
- [x] widgets corners
- [x] loader below the viewport

## How to test it
Go to `/home` in the PR deployment and the Governance section should be testable and the app loaded successfully

### CODE REVIEW

**Comment by DiogoSoaress** [comment]
I've left a comment on the apps loading error fallback https://www.figma.com/file/8nKOe7h90OS7jmn3nksbq9?node-id=405:1857#315622489 and went for this implementation but feedback is welcome @liliiaorlenko 
Screenshot 2022-11-25 at 19 41 04

File: src/components/dashboard/GovernanceSection/GovernanceSection.tsx [hunk 1]
```diff
@@ -24,10 +23,24 @@ const GovernanceSection = () => {
...
-      <Box height="100%" display="flex" flexDirection="column" alignItems="center" justifyContent="center" gap={2}>
-        <LoadIcon fill={`${palette.primary.light}`} />
+      <Box
+        height="100%"
+        display="flex"
+        flexDirection="column"
+        alignItems="center"
+        justifyContent="center"
+        gap={2}
+        maxWidth="80%"
```
Comment (iamacook) [inline src/components/dashboard/GovernanceSection/GovernanceSection.tsx:33]: I'd suggest moving this to the CSS module as well.

File: src/components/safe-apps/AppFrame/index.tsx [hunk 2]
```diff
@@ -198,7 +198,7 @@ const AppFrame = ({ appUrl, allowedFeaturesList, isQueueBarDisabled = false }: A
         <title>Safe Apps - Viewer - {remoteApp ? remoteApp.name : UNKNOWN_APP_NAME}</title>
       </Head>
 
-      <div className={css.wrapper}>
+      <div className={`${css.wrapper} ${isWidget ? css.widgetWrapper : ''}`}>
```
Comment (iamacook) [inline src/components/safe-apps/AppFrame/index.tsx:201]: We should use `classNames` for CSS concatenation like this.

**Review by iamacook — APPROVED** [review]
Looks good to me. I just have a couple small suggestions.

**Comment by liliiaorlenko** [comment]
> I've left a comment on the apps loading error fallback https://www.figma.com/file/8nKOe7h90OS7jmn3nksbq9?node-id=405:1857#315622489 and went for this implementation but feedback is welcome @liliiaorlenko Screenshot 2022-11-25 at 19 41 04

https://www.figma.com/file/8nKOe7h90OS7jmn3nksbq9/Governance-visibility-widgets?node-id=405%3A1857&t=lyBvetC3AE3b6bLp-1
made a design improvements to be consistent with other error states

**Review by schmanu — APPROVED** [review]
Just a small remark. 
All in all the widgets are looking good on the dashboard :)

**Pull request merged**